### DISCUSSION TITLE
Supabase auth-helper nextjs

### DISCUSSION THREAD

**Discussion by mcamendoza1** [supabase/supabase#7457] [Questions]
Have you experience this in supabase auth-helper nextjs 

In my sign-in page, I encounter this `No cookie found!` in my browser console and I'm not printing it out to console.log. 
How to turn it off? or is this a normal/expected behavior? 

![image](https://user-images.githubusercontent.com/12031968/175933254-5681f9b9-4597-4e7f-a66d-db7e6b7e76ca.png)

Here's the code snippet:
```js
// signin.js

// validates if user is signed on 
const { user } = useUser()

useEffect(() => {
  if (user) {
    router.push(`/dashboard/${user.id}`)
  }
}, [user])

async function signIn(data) {
  setIsLoading(true)

  const { user, error } = await supabaseClient.auth.signIn({
    email: data.email,
    password: data.password,
  })

  console.log({ user, error })

  if (error) {
    setSignInError(error.message)
    setIsLoading(false)
    console.log(signInError)
  } else {
    router.push(`/dashboard/${user.id}`)
  }
}
```

https://github.com/supabase-community/auth-helpers/issues/148

**Comment by silentworks** [comment]
This has been fixed in the latest version of the auth-helpers libraries.
